recipe/meta.yaml

```
@@ -1,17 +1,16 @@
 {% set name = "darker" %}
-{% set version = "1.2.2" %}
+{% set version = "1.2.3" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 3a840c2cedec7e4d13c26c450d94f31bd872375450a0c057b9276cc9c2dbad5c
+  sha256: be660bade7045e964a7bd682cd154f2b7e1f7ce9b410bcc5b746f9dddaa14723
 
 build:
-  # noarch: python
-  skip: True  # [win]
+  noarch: python
   number: 0
   script: "{{ PYTHON }} -m pip install . -vv"
```

Could we install `isort` just for tests here?
```suggestion
  script: "{{ PYTHON }} -m pip install '.[isort]' -vv"
```